### DISCUSSION TITLE
"Awesome Semantic UI": discussion

### DISCUSSION THREAD

**Discussion by douglasg14b** [fomantic/Fomantic-UI#2099] [Showroom]
Hi All. I've acquired the [awesome-semantic-ui](https://github.com/douglasg14b/awesome-semantic-ui) repo and am updating it to deprecate deprecated libraries and add the many new and active ones.

I need your help! Point me towards active projects, sites, themes, tools, or even just reference material. Or make a pull request and I'll get it added in.

**Comment by douglasg14b** [comment]
@levithomason 

I see there are three Semantic-React repositories:

* https://github.com/Semantic-Org/Semantic-UI-React
* https://github.com/stardust-ui/react-old (moved to the next repo)
* https://github.com/stardust-ui/react

The `Semantic-UI-React` has seen recent activity as has `stardust-ui/react`. Are these two separate entities at this point or are they one and the same? Or have they forked and are going down different directions? If the later, how would you describe each in a sentence for the "awesome" list?

**Comment by y0hami** [comment]
@douglasg14b They are different frameworks. Semantic-UI-React is the react repo for SUI and is still actively maintained. The stardust repo is the new agnostic framework as discussed in #319 which @levithomason and his team are working on. I wouldn't add it to the list yet since it is still in an early prototyping stage.

**Comment by douglasg14b** [comment]
Will do! Thanks.

**Comment by levithomason** [comment]
Semantic-UI-React is an active React implementation of Semantic UI.  It is "The official React integration."

Stardust is not ready for eyes or attention, as @hammy2899 stated.  There are actually 4 or 5 projects buried in one repo there.  It is a fork of Semantic UI React, converted to typescript, and all concerns being made framework agnostic.  We need time and docs to explain what this is and how it works.

**Comment by prudho** [comment]
I just released a Brunch skeleton powered by Fomantic UI:
- https://github.com/prudho/brunchmantic
